src/applets/clock/ClockApplet.vala

```diff
@@ -301,12 +301,9 @@ public class ClockApplet : Budgie.Applet
         if (!check_date.get_active()) {
             return;
         }
-        string ftime;
```

Comment:
This now breaks vertical panels.

Reply:
The customization via gsettings/dconf keeps possible to use the "<small> tag.
If that is not desirable what do you suggest to do? Should I keep it hardcoded when using vertical panel?